sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -62,6 +66,19 @@ public CompletableResultCode export(Collection<SpanData> spans) {
         return telemetryItemExporter.send(telemetryItems);
     }
 
+    private void addInstrumentationToStatBeat(SpanData span) {
```

Comment:
```suggestion
    private void addInstrumentationToStatsbeat(SpanData span) {
```

Reply:
are you going to turn this into a utility method so that it can be reused for Java agent?

Reply:
I had forgotten this from our meeting discussions. Thanks for the reminder. I suggest to add a static method to the `StatsbeatModule` class. WDYT?

Reply:
[I have added a method](https://github.com/Azure/azure-sdk-for-java/blob/e04961ebbc0f381c46d9597f8d88762efb054d1a/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/FeatureStatsbeat.java#L54) in  `FeatureStatsbeat` to be able to add an instrumentation from a span.